### PR TITLE
Adds changes.

### PR DESCRIPTION
**Pull request by nicovalencia**

### CODE REVIEW

**Review by royriver23 — APPROVED** [review]

File: active-requirements.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ These details describe the requirements to acquire an "active" status each seaso
 
 An Indie must satisfy at least one of the following requirements to be "active" within a season:
 
-- Provide at least 260 billable hours to client projects through the protocol
-- Provide (and document) at least 40 hours of contributions to internal efforts of the DAO
-- Earn at least 2,000 $INDIE during the period of the season
+- Engage in at least 2 billable Indie Protocol sprints for client projects
+- Provide (and document) at least 40 hours of contributions to internal efforts of the DAO, approved and accepted by a Committee Steward
```
Comment (royriver23) [inline active-requirements.md:8]: do you think there could be room to use this as backup if 5,000 INDIE is not reached. just thinking out loud

Comment (nicovalencia) [reply active-requirements.md:8]: Agreed - so any of these can be achieved in order to be considered active (you don't need to satisfy all three)

Comment (royriver23) [reply active-requirements.md:8]: Yup, I guess I asked it wrongly, I was thinking about a possible risk of stretching hours to stay active cause 5000 requirement wasn't met, but I guess we could fallback to the core approval item, then, depending how it goes, we can adjust it next season if needed.

**Review by royriver23 — APPROVED** [review]
niceeeeeeee

File: committee-charters/partnerships-charter.md [hunk 2]
```diff
@@ -6,44 +6,43 @@ _Why does this committee exist?_
...
+- **Structure partnerships** to establish a strong foundation for successful collaborations. This includes defining clear partnership objectives, allocating roles and responsibilities, establishing contractual agreements, outlining deliverables and milestones, facilitating communication and collaboration, and monitoring and evaluating progress.
 
 ### Members
 
-* Nico Valencia 🧢
-* Katy Jeremko
-* Roy Rivera
+- Nico Valencia 🧢
+- Katy Jeremko
+- Roy Rivera
```
Comment (seanconnollydev) [inline committee-charters/partnerships-charter.md:17]: I believe the S10 town hall slides had my name on them, which was a surprise to me. Just want to confirm I'm not expected to be on this committee?

Comment (kayjayko) [reply committee-charters/partnerships-charter.md:17]: Roy should still be the other member on Partnerships (helping with LatAm expansion). Good catch! @nicovalencia can you update this one with the PR?

**Comment by nicovalencia** [comment]
Vote passed, merging!

https://snapshot.org/#/indiedao.eth/proposal/0x2db8dfee4be523ad32c5e0f16207d23f7792ec3217499d304f0ca876c3698e94

**Pull request merged**